src/components/Checkbox/Checkbox.module.css

```diff
@@ -0,0 +1,112 @@
+.wrapper {
```

Comment:
I would suggest to use BEM naming ([as mentioned in the README](https://github.com/Altinn/altinn-design-system#classname-naming-conventions)) ([this is also a great resource on BEM](https://9elements.com/bem-cheat-sheet/?ref=tiny-helpers)), as it conveys the relationship between the classes just by reading them. Currently it is a bit difficult to see the relationship, and I have to toggle between the HTML and styles to get the full picture.

Reply:
I chose this naming style as it is suggested by [the CSS modules documentation](https://github.com/css-modules/css-modules#naming) and [this article on CSS-tricks](https://css-tricks.com/css-modules-part-1-need/#aa-bem-not-required), but I didn't notice that BEM is recommended here. I'll fix.